docs/tasks/debug-application-cluster/core-metrics-pipeline.md

```diff
@@ -0,0 +1,47 @@
+---
+approvers:
+- fgrzadkowski
+- piosz
+title: Core metrics pipeline
+---
+
+Resource usage metrics are important for:
+
+- Kubernetes components to operate (e.g. autoscalers to automatically scale user applications)
+- Kubernetes users to understand their applications behavior
+
+In Kubernetes they are available through Metrics API.
+
+## Metrics API
+
+Metrics API provides current resource usage metrics for pods and nodes. It doesn't offer historical metrics, so one needs to
+archive metrics themselves. In the future Historical API might be defined but it will not be implemented as a part of
+the core metrics pipeline.
+
+The API is discoverable through the same endpoint as the other Kubernetes APIs under `/apis/metrics.k8s.io/` path.
+It also offers the same security, scalability and reliability guarantees.
+
+The API is defined in [k8s.io/metrics](https://github.com/kubernetes/metrics/tree/master/pkg/apis/metrics) repository,
+where the detailed semantic is explained.
```

Comment:
```
Metrics API is defined in [k8s.io/metrics](https://github.com/kubernetes/metrics/tree/master/pkg/apis/metrics) repository.
You can find more information there.
```

Reply:
As a user or an operator, what am I supposed to read from the provided API link? Can we provide some precise, consumable information here instead of pointing the readers to the source code repo?

Reply:
@crassirostris done

@tengqm my goal is to show how the resources look like. Any idea how to do it better?

Reply:
I was looking forward to some concrete examples. Such as "here is an example to show current memory utilization on node A" and "here is another example showing the current CPU utilization of Pod B". We teach users/operators how to use it and give them a taste of the output. I'm afraid not all readers are familiar with Golang and even they do, they have to have some API design knowledge to understand the source code.